### PR TITLE
Gorilla 1 patch

### LINKED ISSUE #119: Gorilla 1 bock & account schema updates
https://github.com/nemtech/catbuffer/pull/35/files

### PR DESCRIPTION
**Pull request by rg911**
- Gorilla 1 patch

Fixed #119

### CODE REVIEW

**Comment by fboucquez** [comment]
Do we need to add the new VRF transactions? @Vektrat, is rest going to retrieve those new transactions?

AccountLinkAction should be renamed to LinkAction according to the catbuffer changes?

**Comment by Vektrat** [comment]
if there are new transactions they need to be added of course, but i've not seen them in the catbuffer changes... do you have any technical links regarding those?

what i see is there were changes in the changes we made, so we will need to update REST https://github.com/nemtech/catbuffer/commit/08777a833b81077aad8e9cc6741d6cd6a0d1fe2c

**Comment by rg911** [comment]
> Do we need to add the new VRF transactions? @Vektrat, is rest going to retrieve those new transactions?
> 
> Should `AccountLinkActionDto` be renamed to `LinkActionDto` according to the catbuffer changes?
> 
> https://github.com/nemtech/symbol-openapi/blob/master/spec/plugins/account_link/schemas/AccountLinkTransactionBodyDTO.yml#L9

Added transactions as well

**Comment by fboucquez** [comment]
I believe there are 2 new transactions

```
    /** Voting key link transaction builder. */
    VOTING_KEY_LINK_TRANSACTION_BUILDER((short) 16707),

    /** Vrf key link transaction builder. */
    VRF_KEY_LINK_TRANSACTION_BUILDER((short) 16963),

```
Steve has added the new open API schemas for those

**Comment by Vektrat** [comment]
yeah already cleared it with him, okay, i was just confused because when i implemented this task those weren't there yet, we will work on them shortly

**Comment by fboucquez** [comment]
Travis is building fine but not reporting to github (again) 
https://travis-ci.com/github/nemtech/catbuffer-generators/builds/164341722

Can anybody merge with admin right?

File: spec/plugins/coresystem/schemas/VotingKeyLinkTransactionBodyDTO.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+type: object
+required:
+  - linkedPublicKey
```
Comment (Vektrat) [inline spec/plugins/coresystem/schemas/VotingKeyLinkTransactionBodyDTO.yml:3]: is this naming correct? shouldn't it be something like `votingKey`?

**Comment by fboucquez** [comment]
Is it possible to improve the documentation on each new transaction and attribute? It will be needed when generating the API doc site and we can then reuse the descriptions in the SDKs' code

File: spec/plugins/coresystem/schemas/VotingKeyLinkTransactionBodyDTO.yml (hunk 1 above)
Comment (Vektrat) [reply spec/plugins/coresystem/schemas/VotingKeyLinkTransactionBodyDTO.yml:3]: well nm, this is what catbuffer says 🤷‍♂️

**Pull request merged**